front-end/src/components/Manager/Record/Package/index.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
```

Comment:
You don't need the Package component here. It is used in Admin Panel to view/edit packages, but you don't have packages in the Manager Panel. Please remove it.